src/test/js/log-karaoke/noStages.js

```diff
@@ -90,7 +90,7 @@ module.exports = {
             this
                 .elements('css selector', 'div.result-item.success', function (collection2) {
                     const count2 = collection2.value.length;
-                    this.assert.notEqual(count, count2);
```

Comment:
yes, the error message is stupid: it says "expected X actual Y" which makes no sense at all for inequality (just carelessness)